### PR TITLE
chore: Rename Graph to EditorGraph

### PR DESCRIPTION
**Pull request by pollend**
I was studying the ScriptGraph logic and was getting confused between Graph and Graph where Graph is used by Editor and Graph used by core. just renamed Graph to match with the source file that its used in not sure how this will affect existing code though.

### CODE REVIEW

**Review by cgalvan — APPROVED** [review]

File: Gems/ScriptCanvas/Code/Editor/Components/EditorGraph.cpp [hunk 1]
```diff
@@ -100,20 +100,8 @@ namespace EditorGraphCpp
...
-        static const char* GetMimeType()
-        {
-            return "application/x-o3de-scriptcanvas";
-        }
-
-        static const char* GetWrappedNodeGroupingMimeType()
-        {
-            return "application/x-03de-scriptcanvas-wrappednodegrouping";
-        }
-    }
```
Comment (lumberyard-employee-dm) [inline Gems/ScriptCanvas/Code/Editor/Components/EditorGraph.cpp:114]: Is there a reason this was moved from the .cpp file to the header?

Comment (pollend) [reply Gems/ScriptCanvas/Code/Editor/Components/EditorGraph.cpp:114]: its in the same namespace as the class EditorGraph.

Comment (lumberyard-employee-dm) [reply Gems/ScriptCanvas/Code/Editor/Components/EditorGraph.cpp:114]: Well it looks like you added it as a static member function to the `EditorGraph` class, so the implementation can still be in the .cpp file, It just would just need to have `EditorGraph::` prepended to the function name

Comment (pollend) [reply Gems/ScriptCanvas/Code/Editor/Components/EditorGraph.cpp:114]: you mean move the declaration into the cpp file?

**Review by lsemp3d — APPROVED** [review]

**Pull request merged**